lib/callbacks.js

```diff
@@ -3,41 +3,33 @@
 const { last } = require('./array');
 const { alias } = require('./utilities');
 
-const falseness = (
-  // Empty function
-  // Returns: boolean, always false
-) => false;
-
-const trueness = (
-  // Empty function
-  // Returns: boolean, always true
-) => true;
-
-const emptiness = (
-  // Empty function
-  // Returns: always undefined
-) => {};
-
-const nop = (
-  // Empty asynchronous callback-last single-argument function
-  callback // function, callback to be called with (null)
-) => {
-  callback(null);
-};
-
-const noop = (
-  // Empty asynchronous callback-last double-argument function
-  empty, // incoming value to be ignored
-  callback // function, callback to be called with (null, null)
-) => {
+// Empty function
+// Returns: <boolean>, always false
+const falseness = () => false;
+
+// Empty function
+// Returns: <boolean>, always true
+const trueness = () => true;
+
+// Empty function
+// Returns: <undefined>
```

Comment:
I am not sure if this should be here since this function doesn't explicitly return `undefined`, it just doesn't return anything.

Reply:
Return value was explicitly stated previously https://github.com/metarhia/common/blob/master/lib/callbacks.js#L18. And since it is supposed to return `undefined`, I think it'll be better to either leave it as it is or add `return undefined` to it.

Reply:
+1 to @belochub.